src/frontend/komponenter/Fagsak/useFagsakApi.ts

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import { useHistory } from 'react-router';
+import { useFagsakDispatch, actions as fagsakActions } from '../FagsakProvider';
+import { apiOpprettBehandling, IOpprettBehandlingData } from '../../api/fagsak';
+import { Ressurs, RessursStatus } from '../../typer/ressurs';
+import { IFagsak } from '../../typer/fagsak';
+import { useOpprettBehandlingContext } from './Opprett/OpprettBehandlingProvider';
+import { Valideringsstatus } from '../../typer/felt';
+
+const useFagsakApi = (
+    settVisFeilmeldinger: (visFeilmeldinger: boolean) => void,
+    settFeilmelding: (feilmelding: string) => void
+) => {
+    const history = useHistory();
+    const [senderInn, settSenderInn] = useState(false);
+
+    const fagsakDispatcher = useFagsakDispatch();
+    const context = useOpprettBehandlingContext();
+
```

Comment:
The function useFagsakApi() is supposed to be reused by all pages, but the context and the opprettBehandling() code below seems to be specific to the OpprettBehandling page. Is this by intention?

Reply:
Yes, each "action" may use context. One could argue that it should be passed as a parameter to the function though 🤔